specification/databricks/resource-manager/Microsoft.Databricks/stable/2018-04-01/databricks.json

```diff
@@ -410,6 +410,14 @@
         "createdDateTime": {
           "description": "Specifies the date and time when the workspace is created.",
           "$ref": "#/definitions/CreatedDateTime"
+        },
+        "workspaceId": {
```

Comment:
adding properties to existing version is breaking change. need to do it in new version

Reply:
@yungezz We're going to backfill all existing databricks workspaces with these 2 properties. Any new workspaces created with this api-version will get these 2 properties by default. And all existing workspaces, we're going to backfill them next week. So this shouldn't technically remain as a breaking change after the backfilling is done. Also, since the backfill process is scheduled for next week, we request you to put a "DoNotMerge" label until we complete the process.

Reply:
HI @amoghnatu, thanks for sharing this. Adding new properties in response will break the GET-PUT pipeline, here's guideline of breaking change https://github.com/Azure/adx-documentation-pr/wiki/Breaking-changes-guidelines#new-property-added-to-response

Reply:
@yungezz Thanks for mentioning this. However, in our case, these 2 new properties are **readonly** and the customer will not be able to set them. Also, these are not available in the template so user cannot set them to provide as input. I have updated the spec to include readonly=true for these 2 properties. Kindly check.